crates/bevy_asset/src/io/embedded/mod.rs

```diff
@@ -15,6 +15,8 @@ use std::path::{Path, PathBuf};
 #[cfg(feature = "embedded_watcher")]
 use alloc::borrow::ToOwned;
 
+/// The name of the `embedded` [`AssetSource`],
+/// as stored in the [`AssetSourceBuilders`] resource.
 pub const EMBEDDED: &str = "embedded";
```

Comment:
What does "embedding" and asset mean? I looked at the example and it didn't really explain it either.

Reply:
It means putting the asset inside the game executable file, in contrast to other assets that are normally loaded from disk to memory when needed. It's normally only used for small engine-related assets so you can use them without having them in your own asset directory.  

It sounds like the example could use a line or 2 describing that.